proto/entity.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto2";
+
+message Interval {
+  required double a = 1;
+  required double b = 2;
+}
+
+message Point {
+  required double x = 1;
+  required double y = 2;
+  required int32 page_index = 3;
```

Comment:
It seems kinda funky to have `page_index` attributes on the `Point` and `BBox` (which is really just `Rectangle`) types. The concepts of point and rectangle are independent of the concept of a page. What do you think about adding a `Rectangle` type and implementing `BBox` as `{page_index: int, rect: Rectangle}`? And something similar for a `PointInPage` type?

Reply:
Makes sense. Will change. Note that this proto file will be deleted shortly (this was just for an experiment)